apps/api/src/interceptors/transform-data-source-in-response/transform-data-source-in-response.interceptor.ts

```diff
@@ -21,7 +21,7 @@ export class TransformDataSourceInResponseInterceptor<T>
   ) {}
 
   public intercept(
-    context: ExecutionContext,
+    _context: ExecutionContext,
```

Comment:
Why?

Reply:
The `_` typically signalizes that we have to declare a variable we don't intend to use. That's also something the typescript compiler understands and the reason why it works.
Another way would be to keep the parameter as is, and then inside the method, assign it to `_`. Something like this:
```ts
public intercept(context, next) {
	_ = context;
	<remaining code>
}
```
I don't think that is cleaner.

Do you think the interceptor will not work properly anymore if the first parameter is not explicitly called `context`?

Reply:
Okay, I didn't know. I would suggest to prefix all unused arguments (where the order matters) with `_` then